web/locales/no-nb.json

```diff
@@ -0,0 +1,275 @@
+        "lowcarbon": "Lavt CO₂-utslipp",
+        "renewable": "Fornybar",
+        "electricity": "Strøm",
+        "electricityproduction": "Strømproduksjon",
+        "electricityconsumption": "Strømforbruk",
+        "bysource": "etter kilde",
+        "emissions": "CO₂-utslipp",
+        "helpfrom": "med hjelp fra",
+        "addeditsource": "<a href=\"%s\" target=\"_blank\">Tilføy eller rediger kilde</a>",
+        "noDataAtTimestamp": "Data midlertidig utilgjengelig for tidspunktet",
+        "noLiveData": "Sanntidsdata er midlertidig utilgjengelig for dette området",
+        "noParserInfo": "Vi har ennå ingen datakilder om området.<br/>Vil du hjelpe med å løse det? Du kan bidra ved å <a href=\"%s\" target=\"_blank\">tilføye regionale data</a>.",
+        "now": "Nå"
+    },
+    "country-history": {
+        "carbonintensity24h": "CO₂-intensitet siste 24 timer",
+        "emissionsorigin24h": "CO₂-utslippskilde siste 24 timer",
+        "emissionsproduction24h": "CO₂-utslipp siste 24 timer",
+        "electricityorigin24h": "Elektrisitetskilde siste 24 timer",
+        "electricityproduction24h": "Strømproduksjonen i de siste 24 timer",
+        "electricityprices24h": "Strømpris siste 24 timer"
+    },
+    "footer": {
+        "foundbugs": "Funnet feil eller har du forbedringsforslag? Tips oss",
+        "here": "her",
+        "faq-text": "Er som er vanskelig å forstå? Se de",
+        "faq": "ofte stilte spørsmål"
+    },
+    "legends": {
+        "windpotential": "Vindpotensial",
+        "solarpotential": "Solpotensial",
+        "colorblindmode": "For fargeblinde",
+        "carbonintensity": "CO₂-intensitet"
+    },
+    "tooltips": {
+        "carbonintensity": "CO₂-intensitet",
+        "crossborderexport": "Eksport til utlandet",
+        "carbonintensityexport": "CO₂-intensitet av eksport",
+        "ofinstalled": "av installeret kapasitet",
+        "utilizing": "utnytter",
+        "withcarbonintensity": "med CO₂-intensitet på",
+        "lowcarbon": "Lavutslipp (CO₂)",
+        "renewable": "Fornybar",
+        "showWindLayer": "Vis vind",
+        "hideWindLayer": "Skjul vind",
+        "showSolarLayer": "Vis sol",
+        "hideSolarLayer": "Skjul sol",
+        "toggleDarkMode": "Skru på nattmodus",
+        "noParserInfo": "Ingen data tilgengelig",
+        "temporaryDataOutage": "Sanntidsdata er midlertidig utilgjengelig",
+        "production": "produksjon",
+        "consumption": "forbruk",
+        "cpinfo": "<b>forbruk</b> inkluderer import og eksport, <b>produksjon</b> tar ikke med import/eksport",
+        "selectLanguage": "Bytt språk",
+        "lowCarbDescription": "Inkluderer fornybar og kjernekraft",
+        "dataIsDelayed": "Data er forsinket"
+    },
+    "misc": {
+        "maintitle": "Sanntids CO₂-utslipp fra strømforbruket",
+        "legend": "Forklaring",
+        "oops": "Ka farsken! Vi får ikke kontakt med serverne. Vi prøver igjen om noen sekund!",
```

Comment:
Haha, is this a common expression in Norway? 😄 
Only thing I could find about it is this link from 2004: https://forum.doktoronline.no/topic/137557-dialekter/page/4/

Reply:
Haha, skidegod linguistic debugging :-)   
It's Northern-Norwegian dialect, and what linguist would regard as a "mild expletive". So the users would be more offended by the server downtime than the use of expletive

Reply:
Haha fair enough, let's keep it like this then :)